lib/active_model/datastore/connection.rb

```diff
@@ -24,13 +24,10 @@ module CloudDatastore
       ENV['GCLOUD_KEYFILE_JSON'] = '{"private_key": "' + ENV['SERVICE_ACCOUNT_PRIVATE_KEY'] + '",
       "client_email": "' + ENV['SERVICE_ACCOUNT_CLIENT_EMAIL'] + '"}'
     end
-  else
```

Comment:
We will need to move the following two lines into the test_helper so that the test suite still works.

Reply:
I've added them to 8169b66.